source/minishift/index.html.erb

```diff
@@ -0,0 +1,183 @@
+---
+title: Minishift - Containerized OpenShift Cluster
+---
+
+<% content_for :header do %>
+<h1 class="animated fadeInDown delay">
+  Minishift
+</h1>
+<p class="animated fadeInUp delay">Develop Applications in a Local Containerized OpenShift Cluster</p>
+<% end %>
+
+<% content_for :navbar do %>
+<div class="nav_logo animated fadeIn">
+  <div class="container">
+    <div class="row">
+      <nav class="col-md-12">
+        <ul class="sf-menu" id="menu">
+          <li>
+            <a href="#about">ABOUT</a>
+          </li>
+          <li>
+            <a href="#getting-started">GET STARTED</a>
+          </li>
+          <li>
+            <a href="#resources">RESOURCES</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+</div>
+<% end %>
+
+<section class="container" id="about">
+  <div class="info_title info_vertical">
+    <div class="row wow info_vertical animated fadeInDown">
+      <div class="col-md-12 text-left">
+        <p>
+          Minishift is a tool that helps you run OpenShift locally by launching a single-node
+          OpenShift cluster inside a virtual machine. With Minishift you can try out OpenShift or
+          develop with it, day-to-day, on your local machine.
+        </p>
+        <p>
+          Minishift uses <a href="https://github.com/docker/machine/tree/master/libmachine">libmachine</a>
+          for provisioning VMs, and <a href="https://github.com/openshift/origin">OpenShift Origin</a>
+          for running the cluster.
```

Comment:
We should also mention that minishift can be run on Microsoft Windows, OS X and GNU/Linux. I know that we have section describing how an user can install in different OSs in the Readme but IMO it would be good to highlight it here.